docs/manual/hacking.rst

```diff
@@ -38,7 +39,7 @@ both backends, specify as arguments to pytest:
     pytest --backend wayland  # Test just Wayland backend
     pytest --backend x11 --backend wayland  # Test both
 
-Testing with the X11 backend requires Xephyr_ in addition to the core
+Testing with the X11 backend requires Xephyr_ and/or xvfb in addition to the core
```

Comment:
Is this an and/or situation? I thought Xephyr was always needed for X tests.

Reply:
you're right. Looks like xvfb just runs xephyr headless